go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/sodafoundation/documentation
 
 go 1.17
+
+require github.com/soda-cdm/documentation v0.0.0-20230525043849-2b985763d78d // indirect
```

Comment:
I think this should be removed. This is to be used for local development.

Reply:
This was also auto generated too...